test/Microsoft.Extensions.MemoryPool.Test/project.json

```diff
@@ -4,13 +4,23 @@
     },
     "dependencies": {
         "Microsoft.Extensions.MemoryPool": "1.0.0-*",
-        "xunit.runner.aspnet": "2.0.0-aspnet-*"
+        "xunit": "2.1.0"
     },
     "commands": {
         "test": "xunit.runner.aspnet"
     },
+    "testRunner": "xunit",
     "frameworks": {
-        "dnx451": { },
-        "dnxcore50": { }
+        "dnx451": {
+            "dependencies": {
+                "xunit.runner.aspnet": "2.0.0-aspnet-*"
+            }
+         },
+        "dnxcore50": {
+            "dependencies": {
+                "Microsoft.NETCore.Platforms": "1.0.0-*",
+                "dotnet-test-xunit": { "version":"0.0.1-*", "target": "package" }
```

Comment:
Tells it to use the package instead of the project if both are available. This is not needed actually, I'll remove it